rs/crypto/test_utils/ni-dkg/src/initial_config/tests.rs

```diff
@@ -294,6 +294,9 @@ fn registry_with_ni_dkg_transcript(
             cup_contents.initial_ni_dkg_transcript_low_threshold =
                 Some(InitialNiDkgTranscriptRecord::from(transcript()));
         }
+        NiDkgTag::HighThresholdForKey(_master_public_key_id) => {
+            unimplemented!("not supported currently")
```

Comment:
Do I understand correctly, these transcripts should go into the `ChainKeyInitialization` introduced with #2337? Then this should already be possible, now that the PR is merged.

Reply:
**TL;DR**: The tests in which the `registry_with_ni_dkg_transcript` helper is used will never be extended so that a `NiDkgTag::HighThresholdForKey` is needed, which is why this is `unimplemented`.

**Background**

This helper function, `registry_with_ni_dkg_transcript`, is used in two tests:
1. `should_correctly_retrieve_initial_low_threshold_ni_dkg_transcript_from_registry`
2. `should_correctly_retrieve_initial_high_threshold_ni_dkg_transcript_from_registry`

Both these tests test `CryptoRegistry::get_initial_dkg_transcripts`, which returns a
```
struct DkgTranscripts {
    low_threshold: NiDkgTranscript,
    high_threshold: NiDkgTranscript,
}
```
(I'm not sure why these tests actually live in `ic-crypto-test-utils-ni-dkg`. I'd say the correct place would be `ic-registry-client-helpers`; we should probably move the tests).

These are the two initial NiDkgTranscripts that a subnet needs to operate (low-threshold for random beacon/tape, high-threshold for state certification). They are required by the ICP protocol itself.

The NIDKG-transcript that is created for vetKeys by means of a key re-sharing request (as part of subnet creation or subnet recovery) is not needed by a subnet to operate. Because of this, such transcripts won't be returned by `CryptoRegistry::get_initial_dkg_transcripts`, but whoever needs them will likely fetch the CUP and then get the transcript out of that.

Because of this, I didn't see a reason

Reply:
Yes, this would indeed help. Done: https://github.com/dfinity/ic/pull/2445/commits/575653378b9093516c7a2da84d7530dad934cad6

With this, I'm resolving the comment. (cc @Sawchord, given that you also commented above).